IPython/html/static/services/kernels/serialize.js

```diff
@@ -75,15 +75,17 @@ define([
         msg = _.clone(msg);
         var offsets = [];
         var buffers = [];
-        msg.buffers.map(function (buf) {
```

Comment:
The logic is the same.  according to my trials on jsperf, `.map` is about 1/4 the speed of an explicit for loop, so I went ahead and changed it.